src/shared/services/payment-info.service.ts

```diff
@@ -0,0 +1,41 @@
+import { BadRequestException, Injectable, NotFoundException } from '@nestjs/common';
+import { GetCryptoPaymentInfoDto } from 'src/mix/models/crypto-route/dto/get-crypto-payment-info.dto';
+import { GetBuyPaymentInfoDto } from 'src/subdomains/core/buy-crypto/route/dto/get-buy-payment-info.dto';
+import { GetSellPaymentInfoDto } from 'src/subdomains/core/sell-crypto/sell/dto/get-sell-payment-info.dto';
+import { AssetService } from '../models/asset/asset.service';
+import { FiatService } from '../models/fiat/fiat.service';
+
+@Injectable()
+export class PaymentInfoService {
+  constructor(private readonly fiatService: FiatService, private readonly assetService: AssetService) {}
+
+  async buyPaymentInfoCheck(dto: GetBuyPaymentInfoDto): Promise<void> {
```

Comment:
I would remove `PaymentInfo` from name, is already included in the service name.